### PR TITLE
17869 - AUTH-WEB - CFS Account Number Added

### PR DESCRIPTION
**Pull request by AbrahamRostampoor**
*Issue #:17869
https://github.com/bcgov/entity/issues/17869

*Description of changes:*
The CFS Account # is added to step 4 of "Make a Payment" page.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2601-2qugxl7x.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2601-2qugxl7x.web.app

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "sbc-auth",
```
Comment (seeker25) [inline package-lock.json:2]: Please remove this file and update package.json, then update package-lock.json by using `npm i`

**Review by seeker25 — CHANGES_REQUESTED** [review]
wrong package-lock.json added, package.json needs to be updated and package-lock.json needs to be updated

**Pull request closed without merging**